artman/cli/main.py

```diff
@@ -417,3 +418,23 @@ def _print_log(pipeline_id):
         '    gcloud beta logging read "logName=projects/vkit-pipeline/logs/%s"'
         % pipeline_id,
     )
+
+
+def _chown_for_artman_output():
+  """Change ownership of artman output if necessary.
+
+  When artman runs in Docker instance, all output files are by default owned by
+  `root`, making it non-editable by Docker host user. When user passes host user
+  id and group id through environment variables via `-e` flag, artman will
```

Comment:
Where is this `-e` flag defined? This makes it sound like it's an artman flag, but I think it's actually a Docker flag. Could you clarify the comment?